src/test/components/TooltipMarker.test.js

```diff
@@ -243,11 +243,37 @@ describe('TooltipMarker', function() {
           },
         },
       ],
+      // This is an old-style bailout marker. This was changed to a Text marker without
+      // a version bump between Gecko profile version 20-21.
       [
         'Bailout_ShapeGuard after getelem on line 3666 of resource://foo.js -> resource://bar.js:3662',
         10,
       ],
+      // This is an old-style invalidation marker. This was changed to a Text marker without
+      // a version bump between Gecko profile version 20-21.
       ['Invalidate http://mozilla.com/script.js:1234', 10],
+      // This is a bailout text marker, as of Gecko profile version 20-21
+      [
+        'Bailout',
```

Comment:
optional: do you think we should also have a marker `BailoutKind::XXX`  (this is what we had since the summer)?

Reply:
I made sure I had one bailout from every profile I shared in the PR description.